docs/analysis.rst

```diff
@@ -10,25 +10,143 @@ the final free energy output that you get. In theory, we could just print out
 "Free energy of Binding =  XX.XXX +- Y.YYY kcal/mol," but that alone does not tell you anything about the quality of
 your free energy estimate. We try to provide guides and built-in analysis to help understand the numbers YANK provides.
 
-The analysis framework comes with a full feature API, command line auto-execution, and
-`Jupyter Notebooks <http://jupyter.org/>`_ to help the user analyze their data programmatically, automatically, or
-visually, depending on their choice.
+The analysis framework comes three ways to help the user analyze their data (use these links to **jump to usage**):
 
-**************
-Analyzing data
-**************
+* :ref:`Command line auto-execution <auto_analyze>`
+* :ref:`Cisually with Jupyter Notebooks<visual_analyze>`
+* :ref:`Full feature API <programmatic_analyze>`
 
-To perform a simple analysis of the data generated by YANK, you can use the ``yank analyze`` facility, even when YANK is running.
 
-.. code-block:: bash
+Analysis Theory
+---------------
+
+We run YANK simulations through  improve the estimate in free energy and other thermodynamic properties. We document them in
+the :doc:`algorithms` page, but summarize them here, and linked to their detailed explanation in the appropriate page.
+When running :ref:`automatically <auto_analyze>`, these steps are run in order to yield the final free energy estimate.
+The analysis procedure is carried out for each phase of the YANK simulation.
+
+
+Equilibration and Decorrelation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We first discard non-equilibrated samples and then subsample to remove correlation effects between the remaining
+samples. The :ref:`timeseries to carry out this analysis from replica exchange <repex_timeseries>` is chosen as the sum
+of potential energies from each sample, evaluated in state it was drawn from. The timeseries is run through the
+``detectEquilibration`` routine of the
+`timeseries module of PyMBAR <https://pymbar.readthedocs.io/en/master/timeseries.html>`_ to determine the correlation
+times. The :ref:`decorrelation rate is analyzed at each point in the timeseries <autocorrelate_algorithm>` by assuming
+all samples before that point are part of the equilibration time, then the number of remaining samples are computed
+by subsampling at the decorrelation rate. The point in the timeseries which preserves the maximum number of samples
+is treated as the sample at which equilibration is complete, and the remaining subsampled data are passed onto MBAR
+for free energy estimate.
+
+
+Replica Mixing Quality
+^^^^^^^^^^^^^^^^^^^^^^
+
+Quality free energy estimates require a sufficient mixing of replicas to
+`improve phase space overlap <http://www.alchemistry.org/wiki/Free_Energy_Fundamentals#Facts_from_the_Free_Energy_Difference_Definition>`_.
```

Comment:
Sufficient mixing does not "improve" phase space overlap; it arises from good phase space overlap.